docs/dev.md

```diff
@@ -164,3 +164,80 @@ commands results in the **LOSS OF DATA**.
     ```shell
     docker-compose down --rmi all -v
     ```
+
+## Checking if there's any documentation with invalid links
+
+The repository comes with documentation spread out over multiple MarkDown files.
+[This workflow file](./../.github/workflows/markdown-link-checker.yml) is set up to check whether there are broken links in
+any of them. If you want to check this locally, you can do so with:
+
+```shell
+# from the repository root directory
+npm install
+npm run mlc
+```
+
+## Running the superlinter locally
+
+We use GitHub's [Super-Linter](https://github.com/github/super-linter) to lint all directories using a variety of
```

Comment:
The CI workflow (.github/workflows/linting.yml) gives less errors then running the commands here. Can a remark be added about the difference?

Reply:
I expanded the note about `act` at the bottom of the section, and made it a bit more prominent under its own H3. I wasn't sure what to do with the `entrypoint` change that apparently is required to make `act` behave -- I ended up just pointing to [your comment](https://github.com/research-software-directory/research-software-directory/pull/624#pullrequestreview-528215446)

Reply:
Great